server/index.js

```diff
@@ -367,6 +367,38 @@ app.get('/api/messages/:userId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/message/convo/:userId/:otherId', (req, res, next) => {
+  let { userId, otherId } = req.params;
+  userId = parseInt(userId, 10);
+  otherId = parseInt(otherId, 10);
```

Comment:
Instead of putting the `userId` in your URL params, have the client include the token and set up some authorization middleware to prevent clients from accessing messages that they aren't supposed to.